rswag-specs/lib/rswag/specs/request_factory.rb

```diff
@@ -194,12 +194,36 @@ def build_form_payload(parameters, example)
 
       def build_json_payload(parameters, example)
         body_param = parameters.select { |p| p[:in] == :body }.first
-        body_param ? example.send(body_param[:name]).to_json : nil
+
+        return nil unless body_param
+
+        raise(MissingParameterError, body_param[:name]) unless example.respond_to?(body_param[:name])
+
+        example.send(body_param[:name]).to_json
       end
 
       def doc_version(doc)
         doc[:openapi] || doc[:swagger] || '3'
       end
     end
+
+    class MissingParameterError < StandardError
+      attr_reader :body_param
+
+      def initialize(body_param)
+        @body_param = body_param
+      end
+
+      def message
+        <<~MSG
+          Missing parameter #{body_param}
```

Comment:
```suggestion
          Missing parameter '#{body_param}'
```

Reply:
I updated with this suggestion & rebased off of a more recent master branch to fix the failing specs